airflow/example_dags/example_time_delta_sensor_async.py

```diff
@@ -36,6 +36,6 @@
     catchup=False,
     tags=["example"],
 ) as dag:
-    wait = TimeDeltaSensorAsync(task_id="wait", delta=datetime.timedelta(seconds=10))
+    wait = TimeDeltaSensorAsync(task_id="wait", delta=datetime.timedelta(seconds=120))
```

Comment:
```suggestion
    wait = TimeDeltaSensorAsync(task_id="wait", delta=datetime.timedelta(seconds=10))
```

Pretty big jump in runtime for an example dag... Maybe leave it as it was, or a more modest increase instead?

Reply:
was an accident, thanks